.evergreen/linker-tests.sh

```diff
@@ -76,15 +92,14 @@ BUILD_DIR="$linker_tests_root/app-cmake-build"
 PREFIX_PATH="$LMC_INSTALL_PATH;$BSON1_INSTALL_PATH"
 SRC_PATH="$linker_tests_deps_root/app"
 run_cmake \
-  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-  $ADDITIONAL_CMAKE_FLAGS \
+  "${common_cmake_args[@]}" \
   -DCMAKE_PREFIX_PATH="$PREFIX_PATH" \
   "-H$SRC_PATH" \
   "-B$BUILD_DIR"
 run_cmake --build "$BUILD_DIR" --target app --config RelWithDebInfo
 
-if [ "$OS_NAME" = "windows" ]; then
-    export PATH="$PATH:$BSON1_INSTALL_PATH/bin:$LMC_INSTALL_PATH/bin"
+export PATH="$PATH:$BSON1_INSTALL_PATH/bin:$LMC_INSTALL_PATH/bin"
+if is_true IS_MULTICONF; then
     APP_CMD="$BUILD_DIR/RelWithDebInfo/app.exe"
```

Comment:
```suggestion
    APP_CMD="$BUILD_DIR/RelWithDebInfo/app$EXE_SUFFIX"
```

Since this is no longer Windows specific.